index.js

```diff
@@ -562,9 +562,9 @@ WA.prototype.getExerciseText = function printExercise (specifier, contentOnly, c
         return callback('The exercise "' + exercise.meta.name + '" is missing a problem definition!')
       }
       if (!contentOnly) {
-        stream.append(exercise.footer, this.options.defaultOutputType) ||
-        stream.append(this.options.footer, this.options.defaultOutputType) &&
-        stream.append('\n')
+        (stream.append(exercise.footer, this.options.defaultOutputType) ||
+          stream.append(this.options.footer, this.options.defaultOutputType)) &&
+          stream.append('\n')
```

Comment:
This small change adds parenthesis around the `||` parts to conform with new rule: http://eslint.org/docs/rules/no-mixed-operators